ramalama/engine.py

```diff
@@ -333,3 +333,10 @@ def add_labels(args, add_label):
     for arg, label_prefix in label_map.items():
         if value := getattr(args, arg, None):
             add_label(f"{label_prefix}={value}")
+
+
+def wait_for_healthy(args):
+    conman_args = ["podman", "wait", "--condition", "healthy", args.name]
```

Comment:
I'm wondering what happens when the container doesn't turn healthy, like
- the container crashes because it cannot correctly access the GPU
- the container blocks forever (for whatever reason around the GPU initialization)
both cases happen to me from time to time 🙃

Reply:
nice, that's a great enhancement, thanks 🙏🏻